web/themes/custom/pineapple/templates/content/node--article--full.html.twig

```diff
@@ -82,7 +82,7 @@
 %}
 {{ attach_library('classy/node') }}
 <article{{ attributes.addClass(classes) }}>
-    <div{{ content_attributes.addClass('node__content', 'u-o-2') }}>
+    <div{{ content_attributes.addClass('article-body', 'u-o-2') }}>
```

Comment:
Remove article body class it's not doing anything, there's no styling against it

Reply:
done